src/App.js

```diff
@@ -1,24 +1,35 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
+import React from "react";
+//import logo from "./logo.svg";
```

Comment:
don'y commit commented code please remove the commented line

Reply:
https://github.com/channabsava/start-learning-react/commit/6409c2bda99dbfbf2558aa35a50e330b57fbae5b